subprojects/plugins/src/main/resources/org/gradle/api/internal/plugins/unixStartScript.txt

```diff
@@ -17,37 +17,97 @@
 #
 
 ##############################################################################
-##
-##  ${applicationName} start up script for UN*X
```

Comment:
This is used to also generate start scripts for Java applications that apply the `application` plugin, so the substitutions need to stay.

Reply:
Done